docs/development/tests.rst

```diff
@@ -23,30 +23,17 @@ environments by running:
 
     tox
 
-By default, tox won't run tests from search,
-in order to run all test including the search tests,
-you need to override tox's posargs.
-If you don't have any additional arguments to pass,
-you can also set the ``TOX_POSARGS`` environment variable to an empty string:
-
-.. prompt:: bash
-
-    TOX_POSARGS='' tox
-
-.. note::
-
-   If you need to override tox's posargs, but you still don't want to run the search tests,
-   you need to include ``-m 'not search'`` to your command:
+To target a specific environment:
 
 .. prompt:: bash
 
-       tox -- -m 'not search' -x
+    tox -e py36
 
-To target a specific environment:
+You can override any pytest's options by running tox like:
 
 .. prompt:: bash
 
-    tox -e py36
+   tox -e py36 -- -m 'not search' -x
```

Comment:
Our `pytest` command already have `-m "not proxito"`. I think that passing another `-m` argument makes this to break. Did you test it? I'm getting this error:

```
root@a1039fd19be4:/usr/src/app/checkouts/readthedocs.org# tox -e py36 -- -m 'not search' -x

...

ERROR: Wrong expression passed to '-m': not: at column 4: expected not OR left parenthesis OR identifier; got end of input

ERROR: InvocationError for command /bin/sh -c ' export DJANGO_SETTINGS_MODULE=readthedocs.settings.test; pytest --cov-report= --cov-config /usr/src/app/checkouts/readthedocs.org/.coveragerc --cov=. --suppress-no-test-exit-code -m "not proxito" -m not' 'search -x' (exited with code 4)
```

Reply:
the next -m will override the former, don't remember if this was working or if I ever used it. Looks like it's ignoring the quotes and reading the command as `-m not  search`

Reply:
I'm not really sure if we should include the search mark to be honest, I think if you want to run just a set of tests you can just use `-k`